acceptance/repository-mongodb/src/__tests__/mongodb.datasource.ts

```diff
@@ -17,7 +17,6 @@ export const MONGODB_CONFIG: DataSourceOptions = {
   database: process.env.MONGODB_DATABASE || 'repository-tests',
 };
 
-export const MONGODB_FEATURES: CrudConnectorFeatures = {
+export const MONGODB_FEATURES: Partial<CrudConnectorFeatures> = {
   idType: 'string',
-  freeFormProperties: false,
```

Comment:
Why is this removed?

Reply:
Probably because the default for freeFormProperties is true? @bajtos , can you confirm? thanks.

Reply:
Yes, `freeFormProperties` is true by default.

I would like connectors to try to support all features, including the ones added in the future. The `options` object should list only flags to disable features not supported by the connector.